<launch>
    <node pkg="orb_slam3" type="mo_rgbd" name="mo_rgbd"
        args="$(find orb_slam3)/Vocabulary/ORBvoc.bin $(find orb_slam3)/METOAK/U60RGBD.yaml"
        output="screen">
        <param name="trajectoryFileLocation" type="string" value="/home/young/vslam_ros/trajectory/" />
        <param name="cam_path" value="/dev/video0" />
        <param name="fps" value="30" />
    </node>
</launch>